Sources/UI/Style/MessageViewStyle.swift

```diff
@@ -282,3 +299,75 @@ extension MessageViewStyle: Hashable {
         hasher.combine(markdownEnabled)
     }
 }
+
+extension MessageViewStyle {
+    struct BackgroundImage {
+        private let images: [Int: UIImage]
+        private let image: UIImage?
```

Comment:
probably better if renamed to `defaultImage`

Reply:
Fixed.